Makefile

```diff
@@ -69,6 +69,7 @@ test: $(SOURCE) VERSION .gobuild
 	@echo Testing inside Docker container for $(GOOS)/$(GOARCH)
 	docker run \
 	    --rm \
+	    -t \
```

Comment:
hivemind: i removed `-ti` as it caused builder to fail during release (because it can't open a tty for reasons) - it should be fine having this on the test target